### ISSUE TITLE
ol_resource date being modified incorrectly

### ISSUE THREAD

**Issue by mikedonnici** [mappcpd/web-services#3]
Suspect `ol_resource.update_at` timestamp is being reset by one of the worker processes. 

At at time of this issue this call to the API for records updated in the last 24 hours...

```http
/v1/a/idlist?t=ol_resource&f=WHERE%20updated_at%20%3E%272017-08-20%2007:01:27%27
# /v1/a/idlist?t=ol_resource&f=WHERE updated_at >'2017-08-20 07:01:27'
```

... return 2049 records.

The first in the (by id asc) was id `2209`, with `updated_at` value of `2017-08-20 12:00:41`.

Then ran this query, which also yielded 2409 records:

```sql
SELECT count(*) FROM ol_resource where updated_at > '2017-08-20 12:00:00';
# 2049
```

Specifying a range + 5mins, yielded 2408:

```sql
SELECT count(*) FROM ol_resource WHERE updated_at > '2017-08-20 12:00:00' AND updated_at < '2017-08-20 12:05:00' ;
```
This is correct as there was one record updated this morning.

So all of the resources were updated within the same 5 minutes, and most likely by the same process.

Important to note is that this does not represent ALL records with id > 2209:

```sql
SELECT count(*) FROM ol_resource where id >= 2209;
# 4411
```

**Comment by mikedonnici** [comment]
pubmedr worker runs at 12:00 UTC and all of these records were added via the pubmedr worker.

This worker fetches all the (same) relevant articles over the past 3 years. If it was doing a blind update then all 4,411 record would have their `updated_at` field set each day. However, there were only 2,408 records updated.

It *may* be possible that the pubmedr worker gets data back that has been updated (at pubmed) and is flagging those as changed?

Need to look more closely at the pubmedr service and see how it decides to update records.

**Comment by mikedonnici** [comment]
After forcing a run of pubmedr can confirm that the resource dates were modified, and then again with another run immediately after.

On check of logs noted this for all records:

```
ResourceByID() record 3957 - could not Parse updated_at parsing time "2015-01-24" as "2006-01-02 15:04:05": cannot parse "" as "15"
Resource 3957 has changed and needs to be updated... SetShortLinkURL(): UPDATE ol_resource SET short_url = "https://csanz.io/r3957" WHERE id = 3957
```

**Comment by mikedonnici** [comment]
Accidentally closed this on commit for #4.

**Comment by mikedonnici** [comment]
After fix for #4 the `updated_at` values for all resources ids >= 2209 were reset to their respective `created_at` values. This was in order the verify that these records where *not* being updated unless there was a significant change in their data.

After a run of all worker services this issue seems to be resolved.
